src/views/home-view.js

```diff
@@ -170,21 +160,6 @@ class HomeView extends ViewMixin(ReduxMixin(PolymerElement)) {
         <img src="../../images/index/integration-gif.gif">
     </div>
 
-    <hr>
-    <div id="fast" class="row">
-        <div class="row-item">
-            <img src="../../images/index/time_saving_r.gif">
-        </div>
-        <div class="row-item">
```

Comment:
I wonder if the text block could remain as it's only the gif that contains the _98%_. Then we wouldn't lose a sales argument.
@toarm, you're the initiator. I guess only you can answer this.

Reply:
I removed it because I thought it looks awkward without the image, but now I can see there is another text block without an image and it looks fine, so I restored the text.